web/src/templates/newsPost.js

```diff
@@ -44,8 +45,9 @@ const Component = (props) => {
     <Layout.FlexWrapper>
       <SEO
         title={data.title}
```

Comment:
Saw that you use sanityProject on the project template, shouldn't we use the same for news and api then? To follow the same pattern

Reply:
Menar du hur jag namnger? Jag är bara lat, kan fixa

Reply:
Menar att vi skiver över `santiyApi/sanityNewsPost` till data på de andra sidorna. Men här gör vi inte det.

Reply:
Fixat!